tests/positionals.rs

```diff
@@ -272,3 +272,21 @@ fn last_positional_second_to_last_mult() {
         .get_matches_from_safe(vec!["test", "tgt", "crp1", "crp2", "--", "arg"]);
     assert!(r.is_ok(), "{:?}", r.unwrap_err().kind);
 }
+
+#[test]
+fn last_positional_var_arg_hyphenized() {
+    let r = App::new("test")
+        .arg(Arg::from_usage("<TARGET> 'some file'"))
+        .arg(Arg::from_usage("[ARGS]... 'some corpus'").allow_hyphen_values(true))
+        .get_matches_from_safe(vec!["test", "-s", "crp2"]);
```

Comment:
You'll need to add something that counts as the `TARGET` arg, "test" is counted as the program name.